shared/common-adapters/text.desktop.tsx

```diff
@@ -115,10 +114,10 @@ function fastGetStyle(
 // Only used by external components
 function externalGetStyle(
```

Comment:
did you check usages of this?

Reply:
I looked for and couldn't find any, IIRC. I was confused at this situation, I would totally believe they're there and I missed them.

Reply:
it's exported as `getStyle`, `search/user-input/index` has one for example. looks like most are in common adapters